src/ig/IGExporter.ts

```diff
@@ -1065,28 +1065,29 @@ export class IGExporter {
 
   /**
    * Adds the package-list.json file to the IG. Generated based on the Configuration history
-   * field.
+   * field, or the package-list.json found at ig-data/package-list.json.
    *
    * @param igPath {string} - the path where the IG is exported to
    */
   addPackageList(igPath: string): void {
-    const packageListPath = path.join(this.igDataPath, '..', 'package-list.json');
-    if (existsSync(packageListPath)) {
-      logger.warn(
-        'A package-list.json file was provided. This file will not be included in SUSHI output. ' +
-          'To create a package-list.json file, define "history" in config.yaml.'
-      );
-    }
+    const packageListPath = path.join(this.igDataPath, 'package-list.json');
+    const isIgDataPackageList = existsSync(packageListPath);
 
     if (this.config.history) {
       const outputPath = path.join(igPath, 'package-list.json');
       outputJSONSync(outputPath, this.config.history, { spaces: 2 });
       logger.info('Generated package-list.json');
       this.updateOutputLog(outputPath, [this.configPath], 'generated');
-    } else if (/^https?:\/\/hl7.org\//.test(this.config.canonical)) {
-      logger.warn(
-        'HL7 IGs must have a package-list.json. Please define "history" in config.yaml to generate one.'
-      );
+      if (isIgDataPackageList) {
+        logger.warn(
+          'A package-list.json file is configured with "history" in config.yaml and provided in ig-data/package-list.json. ' +
+            'Only the package-list configured by config.yaml will be used to generate a package-list.json. ' +
+            'Remove the "history" in config.yaml to use the package-list.json in ig-data/package-list.json.'
```

Comment:
I feel like this message is a little awkward sounding.  I'm definitely biased, but I kind of like the general feel of the one I did for template/ig.ini: https://github.com/FHIR/sushi/blob/f59d3e196e2da19e7f61ec985c2b5c91a14bffcf/src/ig/IGExporter.ts#L986-L989

I won't hold up the PR for this, but I do feel like the message could be improved.

Reply:
I agree, and am changing it currently.

Reply:
Ok in https://github.com/FHIR/sushi/pull/420/commits/e1b96568538bc6b2c3a675224ec51665d0416f93 I made the warning more closely align with the one for template/ig.ini, and I also did the same for the relevant warning for menu/menu.xml, so now all three warnings will be very similar.

Reply:
Oh thank you! I was thinking of doing that also, so I'm glad you did it!